llvm/cmake/modules/AddLLVM.cmake

```diff
@@ -372,6 +374,10 @@ function(set_windows_version_resource_properties name resource_file)
     set(ARG_PRODUCT_NAME "LLVM")
   endif()
 
+  if (NOT DEFINED ARG_PRODUCT_LEGAL_COPYRIGHT)
+    set(ARG_PRODUCT_LEGAL_COPYRIGHT ${LEGAL_COPYRIGHT})
```

Comment:
This will always define `RC_COPYRIGHT` even if no one defines `LEGAL_COPYRIGHT`. Don't think that's an issue though.

Reply:
Should define it to an empty string, which is what it was originally. I'll verify.

Reply:
It used to be undefined, so the ifdef in the RC never got compiled in. Now it always gets compiled. I don't think there's anything bad about that.

Reply:
Verified this happens as expected.